util/jsontemplate.py

```diff
@@ -0,0 +1,81 @@
+import json
+import re
+
+from jsonpath_rw import parse as parse_json_path
+from jsonpath_rw.lexer import JsonPathLexerError
+
+INLINE_PATH_PATTERN = r"\$\{([^}]*)\}"
+
+
+class JSONTemplateParseException(Exception):
+    """ Exception raised if a JSON template could not be parsed. """
+
+
+class JSONTemplate(object):
```

Comment:
Advantages of this versus just using [Jinja.Template](https://jinja.palletsprojects.com/en/2.10.x/api/#jinja2.Template)?

Reply:
Simplicity and safety: This is guaranteed to produce JSON and unlike Jinja, we have no concerns about weird imports or execution of arbitrary code that could exhaust memory, or infinitely loop, etc.